### DISCUSSION TITLE
Why does `pdm init` not create a folder for source code and scripts? What folders should we make for that?

### DISCUSSION THREAD

**Discussion by metaperl** [pdm-project/pdm#1367] [General]
After `pdm init` I have a folder structure, but I did not see in the PDM where I should put my scripts and where I should put my package/libraries.

For the time being, I have created an `ogs_buddy` folder for my package `ogs-buddy` and will put my scripts into `bin`.

**Comment by frostming** [comment]
`pdm init` isn't a scaffolding tool at present, it only creates necessary files for PDM usage.
You will need some project templating tool like `cookiecutter` or `copier`.
